app/components/courses/summary_card_component.html.erb

```diff
@@ -0,0 +1,42 @@
+<li class="app-search-results__item">
+  <%= govuk_summary_card(classes: ["course-summary-card"], title:) do |card| %>
+    <%= card.with_summary_list(actions: false, classes: ["govuk-summary-list--no-border"]) do |summary_list| %>
+      <% summary_list.with_row do |row| %>
+        <% row.with_key(text: location_key) %>
+        <% row.with_value(text: location_value) %>
+      <% end %>
```

Comment:
I have a feeling we can remove the HTML from the ruby code.

```suggestion
      <% summary_list.with_row do |row| %>
        <% row.with_key(text: location_key) %>
        <% row.with_value do %>
          <%= location_value %>
          
          <% if location_hint %>
            <div class="govuk-hint govuk-!-font-size-16"><%= location_hint %></div>
          <% end %>
        <% end %>
      <% end %>
```

It should help with the template visualisation too. Now we now that all fields have a key, a value, and a possible hint.

Reply:
Done!